### ISSUE TITLE
Incorrect JSONCPP_VERSION_PATCH in 1.9.4

### ISSUE THREAD

**Issue by utkarshayachit** [open-source-parsers/jsoncpp#1224]
`version.h` has incorrect value for JSONCPP_VERSION_PATCH for 1.9.4.

**Comment by mathstuf** [comment]
There's not much to be done about the official 1.9.4 release (unless a 1.9.4-1 is something maintainers would consider), but it can be fixed for 1.9.5 at least.

**Comment by axic** [comment]
Would it be possible to make a 1.9.5 release which has a correct version?

In our project we use an assertion like this to verify a correct and supported version:
```
static_assert(
        (JSONCPP_VERSION_MAJOR == 1) && (JSONCPP_VERSION_MINOR == 9) && (JSONCPP_VERSION_PATCH == 3),
        "Unexpected jsoncpp version: " JSONCPP_VERSION_STRING ". Expecting 1.9.3."
);
```

This now matches both 1.9.3 and 1.9.4, yet 1.9.4 will not build correctly as the artefact's name has changed from `libjsoncpp.a` to` libjsoncpp_static.a`.
